src/main/java/uk/gov/justice/hmpps/casenotes/sync/SyncCaseNoteRequest.kt

```diff
@@ -0,0 +1,116 @@
+package uk.gov.justice.hmpps.casenotes.sync
+
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.media.Schema.RequiredMode.REQUIRED
+import jakarta.validation.constraints.NotBlank
+import org.hibernate.validator.constraints.Length
+import uk.gov.justice.hmpps.casenotes.config.Source
+import uk.gov.justice.hmpps.casenotes.notes.AuthoredRequest
+import uk.gov.justice.hmpps.casenotes.notes.TextRequest
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class SyncCaseNoteRequest(
+  @Schema(
+    requiredMode = REQUIRED,
+    example = "1645251",
+    description = "The nomis id for the case note, reflected by for mapping",
+  )
+  val legacyId: Long,
+
+  @Schema(
+    example = "c9475622-676f-4659-8bb5-12a4760280d7",
+    description = "The id for the case note, if provided, the existing case note matching this id will be updated, otherwise a new case note is created.",
+  )
+  val id: UUID?,
+
+  @Schema(
+    requiredMode = REQUIRED,
+    example = "A1234BC",
+    description = "The offender/prison/prisoner/noms number - used to identify the person in prison",
+  )
+  val prisonIdentifier: String,
```

Comment:
Shall we go with `personIdentifier` here to get a bit closer to the `personReference. identifiers` property used in the domain events? That's what I used in the automated case notes endpoint ticket and it gets round this prison(er)Number identifier name issue